com.hoptodesk.HopToDesk.desktop

```diff
@@ -0,0 +1,14 @@
+[Desktop Entry]
+Version=1.0
+Name=HopToDesk
+GenericName=Remote Desktop Software
+Comment=Free Remote Desktop Software. Easily access remote devices securely with end-to-end encryption
+Exec=hoptodesk %u
+Icon=/usr/share/icons/hicolor/128x128/128x128.png
```

Comment:
Icon is wrong, it does not need to be full path, the value should be same as flatpak id.

Reply:
Fixed